### PR TITLE
docs: refactor ip-restriction.md

### PR DESCRIPTION
**Pull request by lunarwhite**
### Description

update document under `docs/zh/latest/plugins/` and `docs/en/latest/plugins/`

### Checklist

- [x] I have explained the need for this PR and the problem it solves
- [x] I have explained the changes or the new features added to this PR
- [ ] I have added tests corresponding to this change
- [x] I have updated the documentation to reflect this change
- [x] I have verified that this change is backward compatible (If not, please discuss on the [APISIX mailing list](https://github.com/apache/apisix/tree/master#community) first)

### CODE REVIEW

File: docs/zh/latest/plugins/ip-restriction.md [hunk 1]
```diff
@@ -23,22 +29,27 @@ title: ip-restriction
 
 ## 描述
 
-`ip-restriction` 可以通过以下方式限制对服务或路线的访问，将 IP 地址列入白名单或黑名单。 单个 IP 地址，多个 IP 地址 或 CIDR 范围，可以使用类似 10.10.10.0/24 的 CIDR 表示法。
+`ip-restriction` 插件可以通过将 IP 地址列入白名单或黑名单来限制对服务或路由的访问。
+
+支持对单个 IP 地址，多个 IP 地址和类似 `10.10.10.0/24` 的 CIDR（Classless InterDomain Routing，无类域间路由）范围的限制。
```
Comment (SylviaBABY) [inline docs/zh/latest/plugins/ip-restriction.md:34]:
```suggestion
支持对单个 IP 地址、多个 IP 地址和类似 `10.10.10.0/24` 的 CIDR（Classless InterDomain Routing，无类域间路由）范围的限制。
```

File: docs/zh/latest/plugins/ip-restriction.md [hunk 2]
```diff
@@ -77,29 +88,62 @@ curl http://127.0.0.1:9080/apisix/admin/routes/1 -H 'X-API-KEY: edd1c9f034335f13
...
+返回 403 HTTP 状态码，代表访问被阻止：
 
 ```shell
-$ curl http://127.0.0.1:9080/index.html -i --interface 127.0.0.2
 HTTP/1.1 403 Forbidden
 ...
 {"message":"Your IP address is not allowed"}
 ```
 
+要更改列入白名单或黑名单的 IP 地址，你只需更新插件配置，无需重启服务：
```
Comment (hf400159) [inline docs/zh/latest/plugins/ip-restriction.md:118]:
```suggestion
如果你需要更改白名单或黑名单的 IP 地址，你只需更新插件配置，无需重启服务：
```

**Review by SylviaBABY — APPROVED** [review]

File: docs/zh/latest/plugins/ip-restriction.md [hunk 3]
```diff
@@ -23,22 +29,27 @@ title: ip-restriction
 
 ## 描述
 
-`ip-restriction` 可以通过以下方式限制对服务或路线的访问，将 IP 地址列入白名单或黑名单。 单个 IP 地址，多个 IP 地址 或 CIDR 范围，可以使用类似 10.10.10.0/24 的 CIDR 表示法。
+`ip-restriction` 插件可以通过将 IP 地址列入白名单或黑名单来限制对服务或路由的访问。
+
+支持对单个 IP 地址、多个 IP 地址和类似 `10.10.10.0/24` 的 CIDR（Classless InterDomain Routing，无类域间路由）范围的限制。
```
Comment (juzhiyuan) [inline docs/zh/latest/plugins/ip-restriction.md:34]:
```suggestion
支持对单个 IP 地址、多个 IP 地址和类似 `10.10.10.0/24` 的 CIDR（无类别域间路由）范围的限制。
```

File: docs/zh/latest/plugins/ip-restriction.md [hunk 4]
```diff
@@ -61,7 +72,7 @@ curl http://127.0.0.1:9080/apisix/admin/routes/1 -H 'X-API-KEY: edd1c9f034335f13
 }'
 ```
 
-当未允许的 IP 访问时，默认返回 `{"message":"Your IP address is not allowed"}`。如果你想使用自定义的 `message`，可以在插件部分进行配置：
+当访问未允许的 IP 时，默认返回 `{"message":"Your IP address is not allowed"}`。如果想使用自定义的 `message`，可以在插件部分进行配置：
```
Comment (juzhiyuan) [inline docs/zh/latest/plugins/ip-restriction.md:75]:
```suggestion
当使用白名单之外的 IP 访问时，默认返回 `{"message":"Your IP address is not allowed"}`。如果想使用自定义的 `message`，可以在插件配置中进行调整：
```

File: docs/zh/latest/plugins/ip-restriction.md [hunk 5]
```diff
@@ -77,29 +88,62 @@ curl http://127.0.0.1:9080/apisix/admin/routes/1 -H 'X-API-KEY: edd1c9f034335f13
 
 ## 测试插件
 
-通过 `127.0.0.1` 访问：
+启用插件后，使用 `curl` 命令先从 IP 地址 `127.0.0.1` 发出请求：
```
Comment (juzhiyuan) [inline docs/zh/latest/plugins/ip-restriction.md:91]:
```suggestion
启用插件后，使用 `curl` 命令访问 APISIX 实例地址：
```

**Review by juzhiyuan — COMMENTED** [review]
Hi, could you please also check the English version?

File: docs/zh/latest/plugins/ip-restriction.md (hunk 3 above)
Comment (lunarwhite) [reply docs/zh/latest/plugins/ip-restriction.md:34]: Hi, "CIDR" is not as common as "IP" or other terms. From the user's perspective and refer to the [doc style guide](https://apisix.apache.org/zh/docs/general/documentation-style-guide/), I prefer to keep the full English name.

Comment (juzhiyuan) [reply docs/zh/latest/plugins/ip-restriction.md:34]: Hi, the full English name doesn't make sense to Chinese readers, right? 😉 Also, if readers want to use the full name, just searching the CIDR.

image

**Review by juzhiyuan — APPROVED** [review]

**Review by SylviaBABY — APPROVED** [review]

**Pull request merged**